python/ecs/ecs-service-with-task-placement/app.py

```diff
@@ -32,10 +32,7 @@
 
 # Create a task definition with placement constraints
 task_definition = ecs.Ec2TaskDefinition(
-    stack, "TaskDef",
-    placement_constraints=[
-        ecs.PlacementConstraint.distinct_instances()
```

Comment:
distinct_instances doesn't belong in task definitions... only member_of is supported